CloudFormation/arkcase.yml

```diff
@@ -1452,16 +1543,530 @@ Resources:
+                Effect: Allow
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-task-execution-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoTaskRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: ecs-tasks.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: AlfrescoTaskPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: secretsmanager:GetSecretValue
+                Resource:
+                  - !GetAtt AlfrescoDatabaseUser.Outputs.UserSecretArn
+                  - !GetAtt Amazonmq.Outputs.AlfrescoUserSecretArn
+                Effect: Allow
+
+              - Action: route53:ChangeResourceRecordSets
+                Resource: !Sub arn:aws:route53:::hostedzone/${PrivateHostedZone}
+                Effect: Allow
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-task-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoTaskDefinition:
+    # NB: Containers in an `awsvpc` task can access each other on `localhost`
+    Type: Custom::TaskDefinition
+    Properties:
+      ServiceToken: !GetAtt TaskDefinitionResourceLambda.Arn
+      family: alfresco
+      cpu: 2048
+      memory: 8192
+      executionRoleArn: !GetAtt AlfrescoTaskExecutionRole.Arn
+      taskRoleArn: !GetAtt AlfrescoTaskRole.Arn
+      requiresCompatibilities: [ FARGATE ]
+      networkMode: awsvpc
+      volumes:
+        - name: alfresco-efs
+          efsVolumeConfiguration:
+            fileSystemId: !Ref AlfrescoEfs
+      containerDefinitions:
+        - name: acs-repository
+          image: !Sub ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/acs-repository:dev4
+          healthCheck:
+            command: [ CMD-SHELL, "curl -f http://localhost:8081/alfresco/" ]
+            interval: 10
+            retries: 2
+            startPeriod: 300  # 5 minutes; Alfresco is very slow to start but 300 seconds is the maximum allowed value
+            timeout: 3
+          # TODO XXX rm if ok to rm
+          # portMappings:
+          #   - containerPort: 8081
+          #     protocol: tcp
+          mountPoints:
+            - sourceVolume: alfresco-efs
+              containerPath: /usr/local/tomcat/alf_data
+          logConfiguration:
+            logDriver: awslogs
```

Comment:
Between Alfresco and Share, they produce a number of log files in different locations (alfresco.log, share.log, catalina.out, some other ones), and a couple of temp folder locations.  Does this `logConfiguration` scoop all these logs into CloudWatch?  Where are the temp files stored?